flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/connectors/hive/TableEnvHiveConnectorTest.java

```diff
@@ -114,6 +114,10 @@ public void testGetNonExistingFunction() throws Exception {
 	public void testDifferentFormats() throws Exception {
 		String[] formats = new String[]{"orc", "parquet", "sequencefile", "csv"};
 		for (String format : formats) {
+			if (format.equals("orc") && hmsClient.getHiveVersion().startsWith("2.0")) {
+				// Ignore orc test for Hive version 2.0.x for now due to FLINK-13998
```

Comment:
Do we plan to test against 2.0.1 as well? If so, we also need to skip parquet and sequencefile tests for 2.0.1.

Reply:
I don't think we can perform test on 2.0.1 unless the maven artifact problem is resolved. However, we can manually test this version with the right artifacts for which parquet and others would be a problem.